nbuild.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Main module of Nest Build: Raven-OS's automated package builder for lazy maintainers.
+"""
+
+import os
+import re
+import importlib.util
+import core
+from core.log import slog, flog
+from dotenv import load_dotenv
+from multiprocessing import cpu_count
+
+
+def main():
+    core.args.parse_args()
+
+    if core.args.get_args().purge:
+        from core.cache import purge_cache
+
+        print("Purging cache... ", end='')
+        purge_cache()
+        print("Done!")
+        exit(0)
+
+    if core.args.get_args().manifest is None:
+        flog("No path to a build manifest given.")
+        exit(1)
+
+    # Clear environment, inflate a default one
+    os.environ.clear()
+
+    # Target & host architecture
+    # TODO FIXME Set as parameter
+    os.environ['TARGET'] = 'x86_64-linux-gnu'
+    os.environ['HOST'] = 'x86_64-linux-gnu'
+
+    # Common flags for the gnu toolchain (cpp, cc, cxx, as, ld)
+    gnuflags = '-O2 -s -m64 -mtune=generic'
+
+    # Pre-processor
+    os.environ['CPP'] = f'{os.environ["TARGET"]}-cpp'
```

Comment:
Shouldn't `TARGET` should be between `'` instead of `"`? There's this error throughout this file, same goes for `HOST`

Reply:
It is not possible to put it in between `'` because of how fstrings works. Yeah, it sucks :/